src/dtflw/lazy_notebook.py

```diff
@@ -66,6 +66,9 @@ def input(self, name: str, file_path: str = None, source_table: str = None) -> L
         source_table: str
             Name of the source table to pass an an input. 
             If 'source_table' is not given then 'name' is used.
+        is_optional: bool
+            Indicator if the input table is optional or not. 
+            If is_optional == True then input table is not necessarily be exist in the file_path.
```

Comment:
If is_optional == True, then the input table may not necessarily exist in the specified file_path.

Reply:
@User1000 Done!